lib/esignatur/user.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+require 'esignatur/api_resource'
+
+module Esignatur
+  # esignatur user representation
+  # More info: https://api.esignatur.dk/Documentation/User
+  class User
+    include ApiResource
+
+    attr_reader :attributes, :user_id, :creator_id, :api
+
+    def initialize(user_id: user, creator_id: creator_id,  attributes: {}, api:)
```

Comment:
this will crash when without `user_id` or `creator_id`. Better:
```suggestion
    def initialize(user_id:, creator_id:,  attributes: {}, api:)
```